src/main/java/meowmeow/MeowmeowException.java

```diff
@@ -0,0 +1,29 @@
+package meowmeow;
+
+/**
+ * <p>Class MeowmeowException is a concrete class that extends the Exception class.</p>
+ * <p>This class is used to create an exception when the user enters an invalid command.</p>
+ * <p>This class is a concrete class because it has an implementation.</p>
+ */
+public class MeowmeowException extends Exception {
+
+    private String message;
+
+    /**
+     * Constructor for MeowmeowException.
+     * @param message the message to be displayed to the user.
+     */
+    MeowmeowException(String message) {
+        this.message = message;
+    }
```

Comment:
Is there a need for these when the superclass already has a message variable?
```suggestion
    /**
     * Constructor for MeowmeowException.
     * @param message The message to be displayed to the user.
     */
    MeowmeowException(String message) {
        super(message);
    }
```

Reply:
I'm not sure, when I try to use the superclass message variable it causes issues with the toString method's return message so I didn't use the superclass variable.